usertrackingsdk/src/main/java/com/visenze/usertrackingsdk/TrackingParams.java

```diff
@@ -0,0 +1,74 @@
+package com.visenze.usertrackingsdk;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ Parameters you want to track in a user event
+ */
+public class TrackingParams {
+    private String action;
+    private String requestId;
+    private String cuid;
+
+    // The mapping of customized tracking parameters
+    // and their corresponding value
+    private Map<String, String> customParams;
+
+    /**
+     * Setup the compulsory tracking parameters
+     * @param action The action of the user event you want to track,
+     *               e.g. "add-to-wish-list", "view", "click"
+     * @param requestId The request id of the current search provided
+     *                  by ViSenze search service
+     */
+    public TrackingParams(String action, String requestId) {
+        this.action = action;
+        this.requestId = requestId;
+        customParams = new HashMap<String, String>();
+    }
+
```

Comment:
I think we can add another constructor which accepts action, requestId and customParams as parameters

Reply:
Added